test/UpdateDeprecations.lean

```diff
@@ -0,0 +1,39 @@
+import Mathlib.Tactic.UpdateDeprecations
+
+run_cmd
+  let fn := "True.hish"
+  let s1 := ".hish and more text"
+  let s2 := "hish and more text"
+  let _s3 := "ish and more text"
+  if findNamespaceMatch fn s1 != some ".hish" then Lean.logInfo "error"
+  if findNamespaceMatch fn s2 != some "hish"  then Lean.logInfo "error"
+  --if findNamespaceMatch fn _s3 != none         then Lean.logInfo "error"
+
+/-- info: true -/
+#guard_msgs in
+#eval
+  let str := "  refine' ⟨_, _⟩"
+  str.replaceCheck "_" "?_" 11 == "  refine' ⟨?_, _⟩"
+
+/-- info: true -/
+#guard_msgs in
+#eval
+  let str := "  refine' ⟨_, _⟩"
+  str.replaceCheck "" "?" 11 == "  refine' ⟨?_, _⟩"
+
+/-- info: true -/
+#guard_msgs in
+#eval
+  let str := "  refine' ⟨_, _⟩"
+  str.replaceCheck "refine'" "refine" 2 == "  refine ⟨_, _⟩"
+
+/-- info: true -/
+#guard_msgs in
+#eval
+  let str := "  refine' ⟨_, _⟩"
+  str.replaceCheck "'" "" 8 == "  refine ⟨_, _⟩"
```

Comment:
These seem to be testing something else, no?

Reply:
These are tests for an "internal function" of `UpdateDeprecations`.

They are based on the script that replaced `refine'` to `refine`, but the main function to test is still the same.